Good-Bids/goodbids
# [#229] Auction Id missing in develop

# Summary

- Prevents app from rendering if auction id is missing

## Issues

- #229

## Testing Instructions

1. View auction screen in develop
2. See that it is correctly requesting data to the right auction id
